MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,244 @@
+	QuizQuestion(
+	  image: "The Godfather",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "The Dark Knight",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "Kill Bill",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "The Avengers",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "Deadpool",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "The Green Knight",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: true
+	),
+	QuizQuestion(
+	  image: "Old",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: false
+	),
+	QuizQuestion(
+	  image: "The Ice Age Adventures of Buck Wild",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: false
+	),
+	QuizQuestion(
+	  image: "Tesla",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: false
+	),
+	QuizQuestion(
+	  image: "Vivarium",
+	  text: "Рейтинг этого фильма больше чем 6?",
+	  correctAnswer: false)
+  ]
+
+
+  private var currentQuestionIndex: Int = 0
+  private var totalGameCount: Int = 0
+
+  private var currentRecord = Record(answersCount: 0, date: Date().dateTimeString)
+
+
+
+  // MARK: - Lifecycle
+  override func viewDidLoad() {
+	super.viewDidLoad()
+	questionImageView.layer.masksToBounds = true
+	showQuestion()
+  }
+
+  @IBAction private func noButtonClicked() {
+	onAnswerButtonClick(false)
+  }
+
+  @IBAction private func yesButtonClicked() {
+	onAnswerButtonClick(true)
+  }
+
+  private func toggleDisableButtons() {
+	yesButton.isEnabled.toggle()
+	noButton.isEnabled.toggle()
+  }
+
+  private func show(quiz step: QuizStepViewModel) {
+	clearResult()
+	questionImageView.image = UIImage(named: step.image)!
+	questionLabel.text = step.text
+	questionCounterLabel.text = step.indexText
+  }
+
+  private func showQuestion() {
+	show(quiz: getQuizStepViewModel())
+  }
+
+  private func showResult() {
+	show(quiz: getResultStepViewModel())
+  }
+
+  private func show(quiz result: QuizResultsViewModel) {
+	let alert = UIAlertController(
+	  title: result.title,
+	  message: result.message,
+	  preferredStyle: .alert
+	)
+
+	let action = UIAlertAction(
+	  title: "Сыграть еще раз",
+	  style: .default, handler: { _ in self.showQuestion() }
+	)
+
+	alert.addAction(action)
+
+	self.present(alert, animated: true, completion: nil)
+  }
+
+  private func showError() {
+	let alert = UIAlertController(
+	  title: "Что-то пошло не так(",
+	  message: "Невозможно загрузить данные",
+	  preferredStyle: .alert
+	)
+
+	let action = UIAlertAction(
+	  title: "Попробовать еще раз",
+	  style: .default, handler: { _ in
+		self.currentQuestionIndex = 0
+		self.showQuestion()
+	  }
+	)
+
+	alert.addAction(action)
+
+	self.present(alert, animated: true, completion: nil)
```

Comment:
```suggestion
	present(alert, animated: true, completion: nil)
```
Используйте самую короткую корректную форму когда это возможно